WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController.m

```diff
@@ -1347,6 +1349,12 @@ - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger
 
 - (void)configureCell:(UITableViewCell *)cell atIndexPath:(NSIndexPath *)indexPath
 {
+    if ([cell isKindOfClass:[BloggingRemindersCell class]]) {
+        cell.accessoryType = UITableViewCellAccessoryNone;
```

Comment:
Can we just `return` here instead, if we never set these for a `BloggingRemindersCell`?